bin/refresh-verify-change.sh

```diff
@@ -34,8 +36,8 @@ need_cmd "curl"
 need_cmd "jq"
 need_cmd "tee"
 
-rdeps_url="https://bldr.habitat.sh/v1/rdeps/core/$changed_plan/group"
-query='.rdeps | .[] | .idents | map(select(startswith("core"))) |select(length > 0) | .[] | ltrimstr("core/")'
+rdeps_url="https://bldr.habitat.sh/v1/rdeps/core/$changed_plan"
```

Comment:
I'm curious about this change.   The rdeps api returns an alphabetical list of plans that should be built,  where the rdeps group api returns the rebuild order grouped by jobs that can be run in parallel.   My thinking was that we should rebuild in the order Builder would use to ensure we're using the correct dependencies through the rebuild process.

Reply:
The group endpoint was 404 for me.

Reply:
Was this with an on-prem depot? That's a relatively new endpoint and may not be available with on-prem yet. 

If not, was it a specific package?  Does https://bldr.habitat.sh/v1/rdeps/core/libyaml/group  work for you?

Reply:
libyaml works.
glibc doesn't work.

https://bldr.habitat.sh/v1/rdeps/core/glibc/group

Reply:
That is probably because we currently have auto-rebuilds turned off for glibc which is unfortunate for the original approach of this script.